app/app.go

```diff
@@ -97,6 +97,8 @@ type TestConfig struct {
 	SimnetBMockOpts []beaconmock.Option
 	// BroadcastCallback is called when a duty is completed and sent to the broadcast component.
 	BroadcastCallback func(context.Context, core.Duty, core.PubKey, core.AggSignedData) error
+	// Proposer is boolean flag to enable/disable block proposer flow
+	Proposer bool
```

Comment:
Sure, I removed this boolean field from here and since we already have `WithDeterministicProposerDuties`, I have used that based on simnet args in test.